src/pages/Main/MainShortCutItem/MainShortCutItem.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MainShortCutItem = ({ img, title }) => {
+  return (
+    <MainShortCutItemWrapper>
+      <MainShortCutItemBox>
+        <img src={img} alt="shortcut" />
```

Comment:
위 코멘트와 마찬가지로 alt text를 수정하면 좋을 것 같아요
```suggestion
        <img src={img} alt={title} />
```